pronotepy/pronoteAPI.py

```diff
@@ -200,12 +200,12 @@ def _parse_html(html: bytes) -> dict:
             HTML attributes
         """
         parsed = BeautifulSoup(html, "html.parser")
-        onload = parsed.find(id='id_body')
-        if onload:
+
+        if onload:= parsed.find(id='id_body'):
```

Comment:
The walrus operator is new in 3.8, we still support 3.7

Reply:
Will it be updated?

Reply:
Maybe sometime in the future... The lib works fine on ver 3.7. I don't think we need the new features right now.

Reply:
Right.